### PR TITLE
Remove duplicate labels from assignments filter

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -**  Remove `<p>`s acting as duplicate label elements in the "Assignments" filter view in gradebook.

JIRA: [EDUCATOR-5502](https://openedx.atlassian.net/browse/EDUCATOR-5502)

**What changed?**

- Removed the following duplicate labels:
    - "Assignment Types:"
    - "Assignment"
    - "Grade Range (0% - 100%)"
- Removed some styling to allow filters to match across filter types

**Developer Checklist**
- [x] Test suites passing
- [x] Received code-owner approving review
- [x] Bumped version number [package.json](../package.json)

**Testing Instructions**

- Run gradebook, verify labels appear as in screenshots and filtering functionality remains

**Before**
Screen Shot 2020-12-15 at 12 42 28

**After**
Screen Shot 2020-12-15 at 12 43 37

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by nsprenkle** [comment]
@jansenk, I got an out-of-band 👍 from @wittjeff. Any other feedback?

**Review by jansenk — APPROVED** [review]
